### ISSUE TITLE
A11y_Model Builder_Evaluate_Predict_Status Messages : After pressing enter on 'Predict' button, screen reader is not announcing updated result details.

### ISSUE THREAD

**Issue by SachinChavhan317** [dotnet/machinelearning-modelbuilder#2684]
### GitHub Tags:
#WCAG4.1.3; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Status Messages; #Narrator; #NVDA; #Win32;

### Environment:
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22621.1702
Narrator
NVDA : Version 2020.4

### Repro Steps:

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. 
7. Complete the required details of All the three(scenario, environment, data, Train) steps.
8. Evaluate screen will appear. Tab till Try You Model section.
9. Tab till "Predict" button and press enter.
10. Turn on screen reader
11. Verify whether after pressing enter on 'Predict' button, screen reader is announcing updated results details or not.

### Actual Result:
After pressing enter on 'Predict' button, screen reader is not announcing updated result details.

**Same issue observed with NVDA**

### Expected Result:
After pressing enter on 'Predict' button, screen reader should be announcing updated results detail.

### User Impact:
Screen reader users will not get predict information if screen reader does not announce updated result.

### Attachment:
Narrator
![Narrator-Presentation1](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/bbac5857-b755-43e7-b7ff-6c35bef25329)

NVDA

![NVDA-Presentation2](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/e1a59b41-e3db-49c6-89f1-babda7da1d1c)

**Comment by zewditu** [comment]
We have results in Grids, how do we make a Grid alive region?

**Comment by zewditu** [comment]
For object detection scenario we have 
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/36615490/d413a0d9-0c84-44f6-a61b-900bd28cc8bd)

**Comment by zewditu** [comment]
@SachinChavhan317 , use [this](https://microsoft.sharepoint.com/sites/PrivateBuildforAccessibility/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents%2FMicrosoft%2EML%2EModelBuilder%2Evsix&parent=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents) build to validate this issue.

**Comment by SachinChavhan317** [comment]
Verified the bug on below version:
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.5

The issue is fixed, hence closing the bug.
Adding attachment for reference. #Closed;
![2684_fixed](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/8525e742-498c-41f7-aa43-ee0f53ff9c1c)
